rpcserver.go

```diff
@@ -5546,16 +5546,26 @@ func marshallTopologyChange(topChange *routing.TopologyChange) *lnrpc.GraphTopol
 
 	nodeUpdates := make([]*lnrpc.NodeUpdate, len(topChange.NodeUpdates))
 	for i, nodeUpdate := range topChange.NodeUpdates {
+		nodeAddrs := make([]*lnrpc.NodeAddress, len(nodeUpdate.Addresses))
```

Comment:
Note that this line will make `len(nodeAddrs) == len(nodeUpdate.Addresses)`. I think you meant:

`make([]*lnrpc.NodeAddresses, 0, len(nodeUpdate.Addresses))`

Reply:
Indeed, thank you!

Reply:
good catch @bhandras, my mistake!